File: code/84236.pas
var i, n, dem, k: integer;
    a : array[1..1000] of integer;
BEGIN
    readln(n);
    for i:= 1 to n do read(a[i]);
    readln(k);
    dem:=0;
    for i:= 1 to n do
        if a[i]=k then inc(dem);
    write(dem)
END.

File: code/84236.cpp
#include <iostream>
using namespace std;
int main()
{
    int i,n, a[1000],k,dem;
    cin>>n;
    for (i=1; i<=n; i++) cin>>a[i];
    cin>>k;
    dem=0;
    for (i=1; i<=n; i++) 
        if (a[i]==k)
            dem++;
    cout<<dem;
    return 0;
}